.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - master
+      - github-actions
+  schedule:
+    - cron: "0 * * * *"
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    env:
+      GH_TOKEN: ${{ SECRETS.GH_TOKEN }}
+      GIT_AUTHOR_NAME: LiteX Robot
+      GIT_AUTHOR_EMAIL: robot@timvideos.us
+      GH_USER: symbiflow-robot
+      GIT_MODE : https
+    steps:
+      - uses: actions/checkout@v2
+
+      # Checkout Repository
+      - name: Checkout
+        run: git fetch --prune --unshallow --tags
+
+      # Install Tools
+      - name: Install Tools
```

Comment:
Maybe use action setup-python instead of doing it manually. See: https://github.com/ghdl/ghdl/blob/master/.github/workflows/Test.yml#L84-L87.